### ISSUE TITLE
Last-token pooling cannot be selected, blocking every LLM-based embedding model

### ISSUE THREAD

**Issue by citron07r** [lemon07r/Vera#92]
## Summary

`LocalEmbeddingPooling` supports only `mean` and `cls`. Last-token pooling cannot be selected by any means: not by `VERA_LOCAL_EMBEDDING_POOLING`, not by `--embedding-pooling`, not by a preset.

That excludes the entire current generation of LLM-based embedding models, all of which pool on the final token.

## Why it matters

`CLAUDE.md` records that any candidate model can be trialled through the `VERA_LOCAL_EMBEDDING_*` overrides without code changes. That is not true for last-token models, so they cannot be benchmarked against the incumbent even informally.

Models currently out of reach:

| model | licence | pooling | notes |
|---|---|---|---|
| `Qwen/Qwen3-Embedding-0.6B` | apache-2.0 | last-token | quantised ONNX via `onnx-community` |
| `codefuse-ai/F2LLM-v2-*` (80M to 14B) | apache-2.0 | last-token | ranks #1 overall on CoIR and MTEB(Code, v1) at 14B; the 330M is the strongest sub-500M model on both |
| `codefuse-ai/C2LLM-*` | apache-2.0 | last-token | tops CoREB |
| `jinaai/jina-code-embeddings-*` | cc-by-nc-4.0 | last-token | no ONNX yet |

It also blocks the default model from being configured correctly, which is filed separately.

## Where it is hard-coded

- `LocalEmbeddingPooling` enum, `Display`, and `FromStr`, including the error text that enumerates valid values.
- The match in `do_embed_once` that selects the pooling branch.
- The clap `value_parser = ["mean", "cls"]` on `--embedding-pooling`, which rejects any other value before `FromStr` is reached.
- The `--embedding-pooling mean|cls` row in `docs/models.md`.

## Implementation note

Pooling must locate the final **unpadded** token, not simply the last column. Counting set bits in the attention mask is correct only under right padding; scanning for the highest set position is correct under both.

**Comment by citron07r** [comment]
Correcting something I overstated in the PR description and in #92.

I wrote that adding last-token pooling "unblocks evaluating Qwen3-Embedding, F2LLM and C2LLM class models". I have now actually run the available exports against Vera's two-input feed instead of reasoning from their file lists, and that claim is too strong.

`onnx-community/Qwen3-Embedding-0.6B-ONNX` and `onnx-community/F2LLM-v2-0.6B-ONNX` are both **causal-LM exports with KV cache**, not encoder exports: 57 required inputs (`input_ids`, `attention_mask`, `position_ids`, and `past_key_values.{0..27}.{key,value}`) and 57 outputs. Both fail at `session.run`:

````
Required inputs (['position_ids', 'past_key_values.0.key', 'past_key_values.0.value', ...])
are missing from input feed (['input_ids', 'attention_mask']).
````

So pooling was one blocker for those models, but not the only one, and removing it does not make them loadable today. What this PR does establish is narrower and still worth having:

- the **default model** is pooled correctly, which is the actual bug and is independently verified
- `last-token` becomes expressible at all, so an encoder-style export of such a model could be trialled without a code change

Nothing in the fix or its tests depends on the claim I am walking back. The correctness evidence is the exact match against the graph's own `sentence_embedding` output, which is unaffected.

Separately, and more usefully: `Alibaba-NLP/gte-modernbert-base` **does** run under Vera's contract. I verified it end to end rather than from the file list: two inputs, single rank-3 `last_hidden_state` output, ran clean on a real tokenized batch. It is apache-2.0, 768-dim so it matches `EMBEDDING_DIM`, cls-pooled so it needs no new pooling mode, and ships `onnx/model_quantized.onnx` (150 MB, self-contained) plus `onnx/model_fp16.onnx` at exactly the paths the loader expects. That is smaller than the current default's 247 MB plus external data file.

On CoIR it scores 0.7110 with full 10-task coverage. The incumbent has run only 2 of the 10 CoIR tasks, so no comparable mean exists for it. I am not proposing a default change here, just recording a verified candidate for #66.
